.github/workflows/main.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  push:
+    branches:
+      - master
+      - stable
+name: Build
+jobs:
+  build:
+    runs-on: macos-latest
+    name: Build and deploy
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/gprbuild
+          path: gprbuild
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/gnatcoll-core
+          path: gnatcoll-core
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/gnatcoll-bindings
+          path: gnatcoll-bindings
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/langkit
+          path: langkit
+          fetch-depth: 0
+      - run: pushd langkit; git checkout ${{ github.ref }} || true; popd
+      - uses: ada-actions/toolchain@b55aaa7
+        with:
+          distrib: community
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - run: |
+          set -x -e
+          export ADALIB_DIR=/tmp/ADALIB_DIR
+          export GPR_PROJECT_PATH=$ADALIB_DIR/share/gpr
+          pip install -r langkit/REQUIREMENTS.dev
+          pip install jsonschema
+          python langkit/setup.py install
+          make -C gprbuild BUILD=production prefix=$ADALIB_DIR GPRBUILD_OPTIONS="-cargs:C -Dst_mtim=st_mtimespec -gargs" libgpr.build.shared libgpr.install.shared
+          make -C gnatcoll-core PROCESSORS=0 prefix=$ADALIB_DIR LIBRARY_TYPES=relocatable build install
+          python gnatcoll-bindings/iconv/setup.py build --reconfigure -j0 --prefix=$ADALIB_DIR --library-types=relocatable
+          python gnatcoll-bindings/iconv/setup.py install
+          export CPATH=/usr/local/include
+          export LIBRARY_PATH=/usr/local/lib
+          python gnatcoll-bindings/gmp/setup.py build --reconfigure -j0 --prefix=$ADALIB_DIR --library-types=relocatable
+          python gnatcoll-bindings/gmp/setup.py install
+          eval `langkit/scripts/manage.sh setenv`
+          ada/manage.py generate -P
+          ada/manage.py build
+          ada/manage.py install $ADALIB_DIR
```

Comment:
I'm confused, are you building LAL relocatable, and only relocatable ? I thought the ALS needed the static one. Are you sure you're not rebuilding LAL in the ALS's GH actions here ?

Reply:
Outdated